index.js

```diff
@@ -1,59 +1,58 @@
-/* jshint node: true */
-'use strict';
-
-var path = require('path');
-var ProjectDataUpdater = require('./lib/project-data-updater.js');
-var FileLockedOperation = require('file-locked-operation');
-var hookshot = require('hookshot');
-var packageInfo = require('./package.json');
-
-module.exports.versionString = function() {
-  return packageInfo.name + ' v' + packageInfo.version;
-};
-
-module.exports.launchServer = function(config) {
-  var lockFilePath = path.resolve(config.workingDir, '.update-lock'),
-      lock = new FileLockedOperation(lockFilePath),
-      buildHook,
-      importHook;
-
-  buildHook = hookshot('refs/heads/' + config.branch, function(info) {
-    var updater,
-        done;
-
-    if (!(info.ref && info.repository.full_name === config.repoFullName)) {
-      return;
-    }
-    updater = new ProjectDataUpdater(config, info.repository, lock);
-    done = logResult('rebuild after update to ' + config.repoFullName);
-    console.log('rebuilding after update to ' + config.repoFullName);
-    return updater.pullChangesAndRebuild().then(done, done);
-  });
-  buildHook.listen(config.buildPort);
-
-  importHook = hookshot('push', function(info) {
-    var updater = new ProjectDataUpdater(config, info.repository, lock),
-        done = logResult(updater.fullName + ' ' +
-          ProjectDataUpdater.ABOUT_YML);
-    return updater.checkForAndImportUpdates(info).then(done, done);
-  });
-  importHook.listen(config.updatePort);
-
-  console.log(module.exports.versionString() +
-    ': Listening on port ' + config.buildPort + ' for push events on ' +
-    config.branch + ' and port ' + config.updatePort +
-    ' for .about.yml updates.');
-};
-
-function logResult(operation) {
-  return function(result) {
-    return Promise(function(resolve, reject) {
-      if (result instanceof Error) {
-        console.error(operation + 'failed: ' + result.message);
-        return reject(result);
-      }
-      console.log(operation + ' succeeded');
-      return resolve(result);
-    });
-  };
-}
+/**
+ * This is a small server that listens for push webhooks from GitHuh repos,
+ *  checks if a target YAML file (like `.about.yml`) has changed,
```

Comment:
Is the extra space of indentation before the word `checks` here intentional? It's fine if it is, just wanted to make sure it wasn't a typo...